src/Microsoft.Health.Dicom.Api/Controllers/ChangeFeedController.cs

```diff
@@ -37,7 +38,7 @@ public ChangeFeedController(IMediator mediator, ILogger<ChangeFeedController> lo
         }
 
         [HttpGet]
-        [ProducesResponseType(typeof(JsonResult), (int)HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(IEnumerable<ChangeFeedEntry>), (int)HttpStatusCode.OK)]
```

Comment:
Below in `GetChangeFeedLatest`, it says it returns `string`, but that's not quite correct either, right? It is really `ChangeFeedEntry`. Are there more examples of this?

Reply:
Oh, that's actually for the `BadRequest` below. Whoops!